profile.html

```diff
@@ -35,7 +35,7 @@ <h1>Quiz App</h1>
       <!-- Card Start -->
       <section>
         <div class="profile">
-          <div class="profile--card">
+          <div class="profile__card">
```

Comment:
I think you could also add the class to your section to reduce the amount of your `div` elements. Maybe some stylings need to be adjusted then. :)